lib/translator/filter.js

```diff
@@ -11,34 +11,27 @@
  * @returns {Object.<string, string|Object>}
  */
 function filterByKey(language, keyFilter) {
```

Comment:
do we have existing tests to support the change ?

Reply:
yes, we do 
https://github.com/bigcommerce/paper/blob/af6e2359eae66dc2925fa344e4a2bff997b6cbe2/spec/lib/translator.js#L215
https://github.com/bigcommerce/paper/blob/af6e2359eae66dc2925fa344e4a2bff997b6cbe2/spec/lib/translator.js#L239